backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/OffeneStruktur.java

```diff
@@ -7,7 +7,7 @@
  * Represents the "OffeneStruktur" of a norm in LDML.de.
  */
 @Getter
-public class OffeneStruktur extends Dokument {
+public non-sealed class OffeneStruktur extends Dokument {
```

Comment:
Do we have any use case for possibly extending this class? And the other two also extending the `sealed` Dokument?

Reply:
We actually only verküdungsfassungen will come in, since the entfwurfsfassung are only used on the EGG side right?